Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:3.2.2
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
Is `nodejs` needed?

Reply:
Thank you for catching this!
`nodejs` was unnecessary, but `libvips` was, so I removed nodejs with ccc4dfa and added `libvips` with 2e70013